infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -378,7 +510,10 @@ public ReplicationStatusVal getReplicationRemainingEntries(String clusterId) {
 
     public void setDataConsistentOnStandby(boolean isConsistent) {
         ReplicationStatusKey key = ReplicationStatusKey.newBuilder().setClusterId(localClusterId).build();
-        ReplicationStatusVal val = ReplicationStatusVal.newBuilder().setDataConsistent(isConsistent).build();
+        ReplicationStatusVal val = ReplicationStatusVal.newBuilder()
+                .setStatus(LogReplicationMetadata.SyncStatus.STOPPED)
```

Comment:
this is only valid for a Standby, isn't it more intuitive if they see the status is ongoing? or do we aim to reflect the status of the replication from an active's point of view? In which case should we add a status that reflects that? cause it can be misleading, thinking that data is not consistent cause replication is stopped, when in reality the active is transferring data.

Reply:
Yes, ongoing makes more sense than stopped. If the user is able to check the status for each standby from the active side, I think empty status might be better.

Reply:
yep, I agree empty status would be better.